libs/utils/utils.js

```diff
@@ -88,16 +88,26 @@ function getEnv(conf) {
   /* c8 ignore stop */
 }
 
+// find out current locale based on pathname and existing locales object from config.
+export function getLocaleFromPath(locales, path) {
+  const split = path.split('/');
+  const localeString = split[1];
+  const locale = locales[localeString] || locales[''];
+  if (localeString === 'langstore') {
+    locale.prefix = split.length >= 3 ? `/${localeString}/${split[2]}` : '';
```

Comment:
The `split.length >= 3` seems like an odd check. Why do we care?

Reply:
It was to avoid /langstore/undefined if url is "https://main--milo--adobecom.hlx.page/langstore"  - but that's not possible anyways since it will 404 given langstore is a folder and not a page unlike in AEM :) Will remove the check.